### DISCUSSION TITLE
Lighthouse false-positive result for missing alt text?

### DISCUSSION THREAD

**Discussion by mold-man1** [GoogleChrome/lighthouse#15714] [Q&A]
Hello everyone,
When testing my product pages on PageSpeed, ( for example https://doyleshamrock.com/product/battery-water-level-indicator/), I get this error shown below. When I hover my cursor over the product image at the top of the page, the alt text is clearly visible, so I don't know why it gets flagged. It happens with all of my product gallery images, and not with any other images on the site.
![image](https://github.com/GoogleChrome/lighthouse/assets/53619362/40696f86-e0b1-44d1-bd27-1e35a533088c)
Is this a Lighthouse issue or something else? Thanks in advance for your help.

**Comment by adamraine** [comment]
Are you referring to the image `title` attribute? Looks like the image `alt` attribute is empty.

**Comment by mold-man1** [comment]
Looking at the page source code, I see that now. Apparently the Product Image feature in WooCommerce is at fault for leaving that field empty instead of inserting my specified alt text. I will reach out to their tech support for help. Thanks for your reply.
